lib/modules/media/media_info_view.dart

```diff
@@ -21,6 +22,22 @@ class MediaInfoView extends StatelessWidget {
   final Media media;
   final ScrollController scrollCtrl;
 
+  Widget _buildFadingHtmlDescription(String description) {
```

Comment:
Can you move this method below the `build` one? Also, remove the 2 comments from it.

Reply:
Sure, I used to do that a lot with the _buildXYZ methods, but probably weird to just have it here 🤷